presto-main/src/main/java/com/facebook/presto/sql/analyzer/Analysis.java

```diff
@@ -101,6 +102,9 @@
     // for describe input and describe output
     private final boolean isDescribe;
 
+    // for recursive view detection
+    private final Stack<Table> tablesForView = new Stack<>();
```

Comment:
`Stack` is one of the old, deprecated collections like `Vector` and `Hashtable`. Use `Deque` and `ArrayDeque` instead.